docs/pages/introduction/browser-support.md

```diff
@@ -0,0 +1,16 @@
+---
+category: Introduction
+page: Browser Support
+---
+
+# Base UI Browser Support
+
+This project supports the browser versions below including newer releases.
+
+- Chrome 58
+- Firefox ESR
+- Safari 10.1
```

Comment:
I would personally love to only support new-ish versions of each, but it feels counter intuitive to supporting android 4.x since it's much older than last 2 versions of these